### ISSUE TITLE
Cannot express BIE in Json with "Include Meta Header" selected

### ISSUE THREAD

**Issue by asdimitriadis** [OAGi/Score#812]
When I tried to express **multiple** BIE with "Include Meta Header" selected nothing happened. No file has been created and downloaded.

**Comment by hakjuoh** [comment]
@asdimitriadis Please explain more details to reproduce this issue.

**Comment by asdimitriadis** [comment]
The cause of the issue is related to the time that the gateway needs to handle a request and the time that the connection between browser and gateway can remain active. If the former is greater than the latter then the connection between the browser and gateway fails. Having the connection failed, the browser cannot receive the package with the requested schemas from the gateway.

So to reproduce the issue, you will need to select multiple BIEs (in my case was 10). 
In other words, you need to select the adequate number of the BIEs so that:
time that gateway needs to generate schema **>** time the connection between browser and gateway can remain active

**Comment by hakjuoh** [comment]
This problem caused during iteratively generating pre-loaded data for each BIE. I've changed it to loading that data for all of them. So the processing time would be faster than before. But, it means that memory resource also would be more consumed. In my opinion, all heavy-workload should put into asynchronous task queue. I'll make an issue for this enhancement.

**Comment by asdimitriadis** [comment]
I am closing this issue as we have scheduled to enhance the BIE expression by presenting asynchronous execution task queue
